### PR TITLE
Adjust wafw00f usage

### PR DESCRIPTION
**Pull request by julianoborba**

### CODE REVIEW

File: nosepass/waf_detector.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from wafw00f.main import WafW00F
+
+
+def detect(url):
+    wafw00f = WafW00F(
+        target=url,
+        proxy='http://10.154.11.143:8888'
```
Comment (resilva87) [inline nosepass/waf_detector.py]: @julianoborba can't you externalize this proxy value?

Comment (julianoborba) [reply nosepass/waf_detector.py]: I was saving for later, but there ya go. Can you check again please?

File: tests/test_waf_detector.py [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 
 
 class WAFDetectorTest(TestCase):
 
-    @patch.object(main.WafW00F, 'identwaf')
+    @patch.object(WafW00F, 'identwaf')
     def test_that_should_detect_waf(self, identwaf):
         identwaf.return_value = ['Cloudflare (Cloudflare Inc.)']
 
-        waf_list = detect('www.vivareal.com.br', 443, True)
```
Comment (carloosamarante) [inline tests/test_waf_detector.py]: This params that you removed, really don't necessary?

File: nosepass/waf_detector.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+from wafw00f.main import WafW00F
+from os.path import abspath, dirname
+import json
+
+NOSEPASS_PATH = abspath(dirname(__file__))
+
+
+def detect(url):
+    with open(f'{NOSEPASS_PATH}/../config.json') as json_data_file:
```
Comment (resilva87) [inline nosepass/waf_detector.py]: @julianoborba you should either use a configuration lib or something like https://github.com/grupozap/rachael/blob/dae14eee9f6688cfb714ca80ff0a0d9b333dd064/app/config.py

File: tests/test_waf_detector.py (hunk 2 above)
Comment (julianoborba) [reply tests/test_waf_detector.py]: Really o/

File: nosepass/waf_detector.py (hunk 3 above)
Comment (julianoborba) [reply nosepass/waf_detector.py]: Once @shikow said about config.py like that, I'm implementing right now

Comment (julianoborba) [reply nosepass/waf_detector.py]: Please check again 2x

File: nosepass/config.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+                      'Safari/537.36'
+    }
+    HTTP_SESSION_PROXIES = {
+        'http': 'http://10.154.11.143:8888',
+        'https': 'http://10.154.11.143:8888'
+    }
+    WAFW00F_PROXY = 'http://10.154.11.143:8888'
+
+
+class ProductionConfig(Config):
```
Comment (carloosamarante) [inline nosepass/config.py]: We are going to use this enviroments right now? I think, for now, you could remove :)

Comment (resilva87) [reply nosepass/config.py]: Best to leave as a template

**Pull request merged**